backend/api/migrations/0006_auto_20180312_1338.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.0.2 on 2018-03-12 13:38
```

Comment:
Arbitrarily added the comment here. Consider squashing migrations so that we don't accrue a million of them. Django is built to handle hundreds at a time, but six migrations can be squashed to a single one for a single PR. People might disagree with me on that one though